modules/framework/galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -289,8 +291,14 @@ private void processCommandLine(String[] args) throws ParseException {
             }
         }
 
-        checkForBootstrap(commandLine);
+        if (!commandLine.hasOption(NO_BOOTSTRAP_OPTION)) {
```

Comment:
No need, change the default behaviour of `checkBootstrap` to initialise a blank Properties object.

Reply:
Done in https://github.com/galasa-dev/galasa/pull/306/commits/7cf616c9bb0f8830b3226cf0b3e5edae1a41367a